tool_collections/bcftools/bcftools_call/bcftools_call.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0"?>
+<tool id="bcftools_call" name="BCFtools call" version="1.0.0">
+    <description>SNPs and indels</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <expand macro="stdio" />
+    <expand macro="version_command" />
+    <command>
+        <![CDATA[
+            @INPUT_FORMAT@
+            bcftools index ${input} &&  
```

Comment:
Input can be one of: format="bcf,bcf_bgzip,vcf,vcf_bgzip"
The indexing step is not required by vcf and for other we can probably use the bcf_index and co.